cmd/vic-machine/common/ops_credentials_test.go

```
@@ -60,4 +60,34 @@ func TestProcessOpsCredentials(t *testing.T) {
 	configureOps.OpsUser = nil
 	err = configureOps.ProcessOpsCredentials(isCreateOp, "", nil)
 	assert.NotNil(t, err)
+
+	// Test correct grant permissions
+	createOps = &OpsCredentials{}
+	createOps.OpsUser = &opsUser
+	createOps.OpsPassword = &opsPassword
+	grantPerms := true
+	createOps.GrantPerms = &grantPerms
+	err = createOps.ProcessOpsCredentials(true, adminUser, &adminPassword)
+	assert.NoError(t, err)
+	assert.Equal(t, *createOps.OpsUser, opsUser)
+	assert.Equal(t, *createOps.OpsPassword, opsPassword)
+	assert.True(t, *createOps.GrantPerms)
+
+	// Create Negative test: grantPerms is set to true but there is no ops-user,
+	// grantPerms should be reset to false
+	createOps = &OpsCredentials{}
+	createOps.OpsUser = nil
+	createOps.OpsPassword = nil
+	grantPerms = true
+	createOps.GrantPerms = &grantPerms
+	err = createOps.ProcessOpsCredentials(true, adminUser, &adminPassword)
+	assert.NoError(t, err)
```

I'm inclined to have this be an error instead of silently ignored.